recipe/meta.yaml

```
@@ -32,15 +32,17 @@ requirements:
     - requests >=2.31
```

```suggestion
    - requests >=2.31
  run_constrained:
    - openapi-core >=0.18.0,<0.19.0
```
https://github.com/jupyterlab/jupyterlab_server/blob/v2.27.3/pyproject.toml#L68